services/analytics-service/app.py

```diff
@@ -53,13 +69,47 @@ def receive_event():
     if 'feedback' not in body:
         return f"'feedback' not in request", 400
 
-    if 'upload_id' not in body:
-        return f"'upload_id' not in request", 400
+    if 'uploadId' not in body:
+        return f"'uploadId' not in request", 400
```

Comment:
I changed to `uploadId` to be consistent with others

Reply:
Yeah, good idea! I've noticed that today :)